hpy/devel/src/runtime/ctx_contextvar.c

```diff
@@ -0,0 +1,17 @@
+#include <Python.h>
+#include "hpy.h"
+
+#ifndef HPY_ABI_CPYTHON
+   // for _h2py and _py2h
+#  include "handles.h"
+#endif
+
+_HPy_HIDDEN int32_t
+ctx_ContextVar_Get(HPyContext *ctx, HPy context_var, HPy default_value, HPy *result)
+{
+    PyObject * py_result = NULL;
+    int32_t ret = PyContextVar_Get(_h2py(context_var), _h2py(default_value), &py_result);
+    if (ret < 0) return ret;
```

Comment:
This branch can just go away. In case `ret < 0` we will have `py_result == NULL` and `_py2h` will correctly convert that to `HPy_NULL` which we need to write to `*result` in case of an error.
You can then also remove the initialization of `py_result = NULL`.

Reply:
OK